mountpoint-s3-client/tests/get_object.rs

```diff
@@ -75,6 +75,32 @@ async fn test_get_object_large() {
     .await;
 }
 
+#[tokio::test]
+async fn test_get_object_one_byte_range() {
+    let sdk_client = get_test_sdk_client().await;
+    let (bucket, prefix) = get_test_bucket_and_prefix("test_get_object_one_byte_range");
+
+    // Create one object named "onebyte"
+    let key = format!("{prefix}/onebyte");
+    let body = b".";
+    sdk_client
+        .put_object()
+        .bucket(&bucket)
+        .key(&key)
+        .body(ByteStream::from(Bytes::from_static(body)))
+        .send()
+        .await
+        .unwrap();
+
+    let client: S3CrtClient = get_test_client();
+
+    let result = client
```

Comment:
As I can see we are not checking for E-Tag here in get_object. Just wanted to know that while testing a feature, we should enter every parameter for it for it being more robust or just the ones relevant for our use case? (In general scenarios)